compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -958,6 +967,31 @@ OMR::CodeCacheManager::chooseCacheStartAddress(size_t repositorySize)
    }
 
 
+void
+OMR::CodeCacheManager::increaseCurrTotalUsedInBytes(size_t size)
+   {
+   self()->decreaseFreeSpaceInCodeCacheRepository(size);
+
+      {
+      UsageMonitorCriticalSection updateCodeCacheUsage(self());
+      _currTotalUsedInBytes += size;
+      _maxUsedInBytes = (_maxUsedInBytes < _currTotalUsedInBytes) ? _currTotalUsedInBytes : _maxUsedInBytes;
```

Comment:
Could we use `std::max()` instead?
```suggestion
      _maxUsedInBytes = std::max(_maxUsedInBytes, _currTotalUsedInBytes);
```